docker/readmodel-db/eservice-template.sql

```diff
@@ -0,0 +1,101 @@
+CREATE SCHEMA IF NOT EXISTS readmodel_eservice_template;
+
+CREATE TABLE IF NOT EXISTS readmodel_eservice_template.eservice_template (
+  id UUID,
```

Comment:
True that this is made not null by the PRIMARY KEY constraint, but being the constraint on a different line it's not so explicit - we could make it NOT NULL explicitly (I think we have the same in all other tables also defined in previous PRs for other entities).
No strong opinion, we could also leave it like this...

```suggestion
  id UUID NOT NULL,
```